Formula/m/mbpoll.rb

```diff
@@ -0,0 +1,29 @@
+class Mbpoll < Formula
+  desc "Command-line utility to communicate with ModBus slave (RTU or TCP)"
+  homepage "https://epsilonrt.fr"
+  url "https://github.com/epsilonrt/mbpoll/archive/refs/tags/v1.5.2.tar.gz"
+  sha256 "7d960cd4459b5f7c2412abc51aba93a20b6114fd75d1de412b1e540cfb63bfec"
+  license "GPL-3.0-only"
+  head "https://github.com/epsilonrt/mbpoll.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libmodbus" => ["3.1.4"]
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+    bin.install "build/mbpoll"
```

Comment:
Is this not installed by `--install`?

Reply:
you're right.   I removed the `bin.install "build/mbpoll"`